integration/js/app/video_test/server.js

```diff
@@ -0,0 +1,120 @@
+    const requestUrl = url.parse(request.url, true);
+    if (request.method === 'GET' && requestUrl.pathname === '/') {
+      // Return the contents of the index page
+      respond(response, 200, 'text/html', indexPage);
+    } else if (process.env.DEBUG && request.method === 'POST' && requestUrl.pathname === '/join') {
+      // For internal debugging - ignore this.
+      respond(response, 201, 'application/json', JSON.stringify(require('./debug.js').debug(requestUrl.query), null, 2));
+    } else if (request.method === 'POST' && requestUrl.pathname === '/join') {
+      if (!requestUrl.query.title || !requestUrl.query.name || !requestUrl.query.region) {
+        throw new Error('Need parameters: title, name, region');
+      }
+
+      // Look up the meeting by its title. If it does not exist, create the meeting.
+      if (!meetingTable[requestUrl.query.title]) {
+        meetingTable[requestUrl.query.title] = await chime.createMeeting({
+          // Use a UUID for the client request token to ensure that any request retries
+          // do not create multiple meetings.
+          ClientRequestToken: uuidv4(),
+          // Specify the media region (where the meeting is hosted).
+          // In this case, we use the region selected by the user.
+          MediaRegion: requestUrl.query.region,
+          // Any meeting ID you wish to associate with the meeting.
+          // For simplicity here, we use the meeting title.
+          ExternalMeetingId: requestUrl.query.title.substring(0, 64),
+        }).promise();
+      }
+
+      // Fetch the meeting info
+      const meeting = meetingTable[requestUrl.query.title];
+
+      // Create new attendee for the meeting
+      const attendee = await chime.createAttendee({
+        // The meeting ID of the created meeting to add the attendee to
+        MeetingId: meeting.Meeting.MeetingId,
+
+        // Any user ID you wish to associate with the attendeee.
+        // For simplicity here, we use a random id for uniqueness
+        // combined with the name the user provided, which can later
+        // be used to help build the roster.
+        ExternalUserId: `${uuidv4().substring(0, 8)}#${requestUrl.query.name}`.substring(0, 64),
+      }).promise()
+
+      // Return the meeting and attendee responses. The client will use these
+      // to join the meeting.
+      respond(response, 201, 'application/json', JSON.stringify({
+        JoinInfo: {
+          Meeting: meeting,
+          Attendee: attendee,
+        },
+      }, null, 2));
+    } else if (request.method === 'POST' && requestUrl.pathname === '/end') {
+      // End the meeting. All attendee connections will hang up.
+      await chime.deleteMeeting({
+        MeetingId: meetingTable[requestUrl.query.title].Meeting.MeetingId,
+      }).promise();
+      respond(response, 200, 'application/json', JSON.stringify({}));
+    } else if (request.method === 'GET' && requestUrl.pathname === '/fetch_credentials') {
```

Comment:
You dont need this. This is for messaging session.

Reply:
Okay, removing this condtion